src/ResourceManager/LogicApp/Commands.LogicApp/Cmdlets/IntegrationAccount/GetAzureIntegrationAccountCallbackUrlCommand.cs

```diff
@@ -36,9 +36,8 @@ public class GetAzureIntegrationAccountCallbackUrlCommand : LogicAppBaseCmdlet
 
         [Parameter(Mandatory = true, HelpMessage = "The integration account name.",
             ValueFromPipelineByPropertyName = true)]
-        [Alias("ResourceName")]
         [ValidateNotNullOrEmpty]
-        public string Name { get; set; }
+        public string IntegrationAccountName { get; set; }
```

Comment:
@pankajsn breaking change, revert this and add a warning that this will be change in future release.
Also update the upcoming breaking change file to document this breaking change coming up in future release.